src/txkube/test/test_model.py

```diff
@@ -373,7 +374,10 @@ class MemoryResponse(object):
 
     def deliverBody(self, protocol):
         protocol.makeConnection(None)
-        protocol.dataReceived(self._body)
+        body = self._body
+        if isinstance(body, unicode):
+            body = body.encode("ascii")
```

Comment:
Having unicode for the body here seems like a bug in application code somewhere.  You don't get unicode out of an http response body.

Reply:
This line:

```
body = dumps(v1_5_model.iobject_to_raw(v1_5_model.v1.Status()))
```

assigns a unicode value to body on Python 3.